lib/modules/manager/regex/readme.md

```diff
@@ -6,6 +6,8 @@ This manager is unique in Renovate in that:
 - Through the use of the `regexManagers` config, multiple "regex managers" can be created for the same repository
 - It can extract any `datasource`
 
+Check our [additional Handlebars helpers](https://docs.renovatebot.com/templates/#additional-handlebars-helpers).
```

Comment:
Maybe needs some context? Eg these apply to xTemplate fields

Reply:
I think it would also be helpful to add a link to the general configuration options. https://docs.renovatebot.com/configuration-options/#regexmanagers